queries/dailyVolumeByDate_test.go

```diff
@@ -98,113 +115,113 @@ func TestDailyVolumeByDate_QueryRow(t *testing.T) {
 		},
 	}
 
-	for _, k := range testCases {
-		t.Run(strings.Replace(fmt.Sprintf("%v", k.queryByOptionalAccountIDs), " ", "_", -1), func(t *testing.T) {
-			// setup db
-			yesterday, _ := time.Parse(time.RFC3339, "2020-01-20T15:00:00Z")
-			today, _ := time.Parse(time.RFC3339, "2020-01-21T15:00:00Z")
-			tomorrow, _ := time.Parse(time.RFC3339, "2020-01-22T15:00:00Z")
-			setupStatements := []string{
-				kelpdb.SqlTradesTableCreate,
-				"ALTER TABLE trades DROP COLUMN IF EXISTS account_id",
-				"ALTER TABLE trades DROP COLUMN IF EXISTS order_id",
-				kelpdb.SqlTradesTableAlter1,
-				kelpdb.SqlTradesTableAlter2,
-				"DELETE FROM trades", // clear table
-				fmt.Sprintf(kelpdb.SqlTradesInsertTemplate,
-					"market1",
-					"1",
-					yesterday.Format(postgresdb.TimestampFormatString),
-					model.OrderActionSell.String(),
-					model.OrderTypeLimit.String(),
-					0.10,  // price
-					100.0, // volume
-					10.0,  // cost
-					0.0,   // fee
-					"accountID1",
-					"",
-				),
-				fmt.Sprintf(kelpdb.SqlTradesInsertTemplate,
-					"market1",
-					"2",
-					today.Format(postgresdb.TimestampFormatString),
-					model.OrderActionSell.String(),
-					model.OrderTypeLimit.String(),
-					0.11,  // price
-					101.0, // volume
-					11.11, // cost
-					0.0,   // fee
-					"accountID1",
-					"oid1",
-				),
-				fmt.Sprintf(kelpdb.SqlTradesInsertTemplate,
-					"market1",
-					"3",
-					today.Add(time.Second*1).Format(postgresdb.TimestampFormatString),
-					model.OrderActionSell.String(),
-					model.OrderTypeLimit.String(),
-					0.12, // price
-					6.0,  // volume
-					0.72, // cost
-					0.10, // fee
-					"accountID1",
-					"",
-				),
-				fmt.Sprintf(kelpdb.SqlTradesInsertTemplate,
-					"market1",
-					"4",
-					tomorrow.Format(postgresdb.TimestampFormatString),
-					model.OrderActionSell.String(),
-					model.OrderTypeLimit.String(),
-					0.12,  // price
-					102.0, // volume
-					12.24, // cost
-					0.0,   // fee
-					"accountID1",
-					"",
-				),
-				fmt.Sprintf(kelpdb.SqlTradesInsertTemplate,
-					"market1",
-					"5",
-					tomorrow.Add(time.Second*1).Format(postgresdb.TimestampFormatString),
-					model.OrderActionBuy.String(),
-					model.OrderTypeLimit.String(),
-					0.12,  // price
-					102.0, // volume
-					12.24, // cost
-					0.0,   // fee
-					"accountID1",
-					"",
-				),
-				// add an extra one for accountID2
-				fmt.Sprintf(kelpdb.SqlTradesInsertTemplate,
-					"market1",
-					"6",
-					today.Format(postgresdb.TimestampFormatString),
-					model.OrderActionSell.String(),
-					model.OrderTypeLimit.String(),
-					0.10,  // price
-					100.0, // volume
-					10.0,  // cost
-					0.0,   // fee
-					"accountID2",
-					"",
-				),
-			}
-			db := connectTestDb()
-			defer db.Close()
-			for _, s := range setupStatements {
-				_, e := db.Exec(s)
-				if e != nil {
-					panic(e)
-				}
-			}
+	// setup db
+	yesterday, _ := time.Parse(time.RFC3339, "2020-01-20T15:00:00Z")
```

Comment:
was this a direct copy paste or did something change inside this too?

Reply:
Direct copy paste - nothing changed, just a small performance fix. 

Previously, the tests were setting up a brand new database in each iteration of the for-loop / for each test case. I found that this was making the run-time substantially slower. I moved the setup steps out of the loop and made it about 5x faster. (.124s vs .6s). I don't think this is happening on the CI, but it's a solid saving without losing any correctness.

Reply:
yes, good fix! 👍